examples/process_script.py

```diff
@@ -1,142 +1,192 @@
-    trajectory_file = 'OIB-P3_20170327_F2004_DGS-INS_RAPID_DGS.txt'
-    # NW Coast Parallel
-    begin_line = datetime(2017, 3, 27, 15, 35)
-    end_line = datetime(2017, 3, 27, 16, 50)
-    gps_fields = ['mdy', 'hms', 'lat', 'long', 'ortho_ht', 'ell_ht', 'num_stats', 'pdop']
-
+from dgp.lib.plots import timeseries_gravity_diagnostic, mapplot_segment, read_meterconfig
+
+# Runtime Options
+write_out = True
+make_plots = True
+add_map_plots = False
+diagnostic = True
+
+# Read YAML config file
+proj_path = os.path.abspath(os.path.dirname(__file__))
+try:
+    with open(os.path.join(proj_path, 'config_runtime.yaml'), 'r') as file:
+        config = yaml.safe_load(file)
+        print('Read YAML configuration file for {}'.format(config['flight']))
+except Exception as e:
+    print('Error reading the config file')
+    # TODO What to do if exception is reached? Exit, or read in test data?
+campaign = config['campaign']
+flight = config['flight']
+begin_line = datetime.strptime(config['begin_line'], '%Y-%m-%d %H:%M')
+end_line = datetime.strptime(config['end_line'], '%Y-%m-%d %H:%M')
+gravity_directory = config['gravity_dir']
+gravity_file = config['gravity_file']
+trajectory_source = config['trajectory_src']
+trajectory_directory = config['trajectory_dir']
+trajectory_file = config['trajectory_file']
+gps_fields = config['gps_fields']
+outdir = config['outdir']
+try:
+    QC_segment = config['QC1']  #
+    QC_plot = True
+except KeyError:
+    print('No QC Segments...')
+    QC_plot = False
+if trajectory_source == 'Waypoint':
+    trajectory_engine = 'c'
+    trajectory_delim = ','
 else:
-    print('Scotia?')
+    trajectory_engine = 'python'
+    trajectory_delim = '\s+'
 
 # Load Data Files
 print('\nImporting gravity')
-gravity = read_at1a(os.path.join(basedir, gravity_directory, gravity_file), interp=True)
+gravity = read_at1a(os.path.join(gravity_directory, gravity_file), interp=True)
+print("Gravity START: {0}".format(gravity.index[0]))
+print("Gravity END:   {0}".format(gravity.index[-1]))
 print('\nImporting trajectory')
-trajectory = import_trajectory(os.path.join(basedir, trajectory_directory, trajectory_file),
-                               columns=gps_fields, skiprows=1, timeformat='hms')
+trajectory = import_trajectory(os.path.join(trajectory_directory, trajectory_file),
+                               columns=gps_fields, skiprows=1,
+                               timeformat='hms', engine=trajectory_engine, delim=trajectory_delim)
 
 # Read MeterProcessing file in Data Directory
-config_file = os.path.join(basedir, gravity_directory, "MeterProcessing.ini")
+config_file = os.path.join(gravity_directory, 'DGS_config_files', 'MeterProcessing.ini')
 k_factor = read_meterconfig(config_file, 'kfactor')
 tie_gravity = read_meterconfig(config_file, 'TieGravity')
-print('{0} {1}'.format(k_factor, tie_gravity))
-flight = gravity_file[4:11]
+print('K-factor:    {0}\nGravity-tie: {1}\n'.format(k_factor, tie_gravity))
```

Comment:
Same here.  Only if you're transposing them do you really need to uses the indexes in the brackets.  Or better yet, make these two prints.